client/src/App.js

```diff
@@ -1,21 +1,41 @@
-import React, { useReducer } from "react";
+import React, { useReducer, useEffect } from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 import reducer from "./reducers";
 import initState from "./initState";
-
+import axios from "axios";
 import { userContext } from "./userContext";
 import { theme } from "./themes/theme";
 import { ProtectedRoute } from "./components/ProtectedRoute";
 import Login from "./pages/Login";
 import SignUp from "./pages/SignUp";
-import OnBoard from './pages/OnBoard';
+import OnBoard from "./pages/OnBoard";
 
 import "./App.css";
 
 function App() {
   const [state, dispatch] = useReducer(reducer, {}, initState);
 
+  //Getting user:
+  useEffect(() => {
+    const getUser = async () => {
+      //dispatch({ type: "FETCH_USER" }); // Why do this if we can init with loading: true?
```

Comment:
We can get away with adding `loading: true` to the initial state since you used an effect hook. However, we will still need a `type: "FETCH"` dispatch for other API calls. I am hesitant to initialize the app in an initial state that is potentially locked up (especially since we'll likely deploy on a free tier service that will require a few moments to spin up a container before handling our request.)